nautobot/docs/release-notes/version-2.4.md

```diff
@@ -166,6 +166,74 @@ As Python 3.8 has reached end-of-life, Nautobot 2.4 requires a minimum of Python
+- [#7107](https://github.com/nautobot/nautobot/issues/7107) - Refactored JobHook model related UI views to use `NautobotUIViewSet` and `UI component framework`.
+- [#7126](https://github.com/nautobot/nautobot/issues/7126) - Refactored CustomLink model related UI views to use `NautobotUIViewSet` and `UI component framework`.
+- [#7141](https://github.com/nautobot/nautobot/issues/7141) - Refactored PowerPanel model related UI views to use `NautobotUIViewSet` and `UI component framework`.
+- [#7148](https://github.com/nautobot/nautobot/issues/7148) - Refactored RackGroup model related UI views to use `NautobotUIViewSet` and `UI component framework`.
+- [#7153](https://github.com/nautobot/nautobot/issues/7153) - Refactored SupportedDataRate model related UI views to use `UI component framework`.
+- [#7155](https://github.com/nautobot/nautobot/issues/7155) - Refactored RackReservation model related UI views to use `NautobotUIViewSet`.
+- [#7158](https://github.com/nautobot/nautobot/issues/7158) - Refactored CloudNetwork model related UI views to use `UI component framework`.
+- [#7159](https://github.com/nautobot/nautobot/issues/7159) - Refactored ComputedField model related UI views to use `NautobotUIViewSet`.
+- [#7162](https://github.com/nautobot/nautobot/issues/7162) - Refactored CloudService model related UI views to use `UI component framework`.
+- [#7172](https://github.com/nautobot/nautobot/issues/7172) - Refactored PowerFeed model related UI views to use `NautobotUIViewSet`.
+- [#7173](https://github.com/nautobot/nautobot/issues/7173) - Refactored JobQueue model related UI views to use `UI component framework`.
+- [#7175](https://github.com/nautobot/nautobot/issues/7175) - Refactored JobButton model related UI views to use `UI component framework`.
+- [#7184](https://github.com/nautobot/nautobot/issues/7184) - Refactored RadioProfile model related UI views to use `UI component framework`.
+- [#7187](https://github.com/nautobot/nautobot/issues/7187) - Add Upstream testing for next/next-3.0 in Apps.
+- [#7187](https://github.com/nautobot/nautobot/issues/7187) - Removes Upstream testing for ltm-1.6.
+- [#7189](https://github.com/nautobot/nautobot/issues/7189) - Updated `mkdocs-material` documentation dependency to `~9.6.12`.
+- [#7189](https://github.com/nautobot/nautobot/issues/7189) - Updated `mkdocs-section-index` documentation dependency to `~0.3.10`.
+- [#7204](https://github.com/nautobot/nautobot/issues/7204) - Refactored SoftwareVersion model related UI views to use `UI component framework`.
+- [#7212](https://github.com/nautobot/nautobot/issues/7212) - Refactored SoftwareImageFile model related UI views to use `UI component framework`.
+- [#7230](https://github.com/nautobot/nautobot/issues/7230) - Modified the mkdocs theme to load Google Tag Manager with the appropriate code for site analytics.
```

Comment:
I have a belated concern about this one - see thread on internal Slack. We might need to back it out and revisit.

Reply:
Reverted